.github/workflows/build-master.yml

```diff
@@ -26,7 +26,8 @@ jobs:
           packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
           packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          ./gradlew publish
+            ./gradlew publish
+            ./gradlew codeCoverageReport
```

Comment:
Are we going to use two steps? I see this is different from [what GraphQL does.](https://github.com/ballerina-platform/module-ballerina-graphql/pull/98) Is there a reason for that?

Reply:
yes. There is a difference. GraphQL uses the xml generated by testerina so we don't need a seperate task there. In http we can't use that xml because that xml only contains the `http-ballerina-tests` module. so we need a task to generate xml which includes `http-ballerina` module.